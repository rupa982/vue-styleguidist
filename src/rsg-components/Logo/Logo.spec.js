import React from 'react';
import LogoRenderer from './LogoRenderer';

it('renderer should render header', () => {
	const actual = render(<LogoRenderer>Vue Styleguidist</LogoRenderer>);

	expect(actual).toMatchSnapshot();
});
