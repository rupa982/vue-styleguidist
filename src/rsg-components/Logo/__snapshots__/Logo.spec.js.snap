// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer should render header 1`] = `
<h1>
  &lt;button&gt;OK&lt;/button&gt;
</h1>
`;
